### ISSUE TITLE
Can't remove opponents or players (chrome)

### ISSUE THREAD

**Issue by RaySSharma** [RaySSharma/foundry-modules#1]
Can't remove opponents or players from calculation or form on Chrome.

```
builder-form.js:230 Uncaught TypeError: Cannot read property 'className' of undefined
    at EncounterBuilderApplication._onDragEnd (builder-form.js:230)
_onDragEnd @ builder-form.js:230
```

**Comment by RaySSharma** [comment]
This is tied to the fact that `event.explicitOriginalTarget` is a Firefox-only attribute that shouldn't be used for cross-browser compatibility. Needs a workaround!

**Comment by RaySSharma** [comment]
Solved by #4
